modules/contact/controllers/add.controller.js

```diff
@@ -0,0 +1,44 @@
+const Joi = require('joi');
+const addContact = require('../services/add.service');
+
+module.exports = async (req, res) => {
+  try {
+    {
+      // validation
+      const expectedBody = Joi.object()
+        .required()
+        .keys({
+          userId: Joi.string().required(),
+          name: Joi.string().required(),
+          email: Joi.string().required(),
+          phones: Joi.array().items(Joi.string().required())
+        });
+
+      const validation = Joi.validate(req.body, expectedBody);
+      if (validation.error)
+        return res
+          .status(500)
+          .json({ success: false, message: validation.error, data: null });
+    }
+    try {
+      await addContact(req.body);
```

Comment:
Sending full req.body is a risk, you should extract only the required properties